### ISSUE TITLE
Holder loses sign in the middle in certain circumstances

### ISSUE THREAD

**Issue by Mmlh1** [cigumo/krvi#511]
https://github.com/user-attachments/assets/9c786c4e-9fba-4a9e-9107-54375251547e

Slowed down version:

https://github.com/user-attachments/assets/5a6eec9c-566d-44b2-848e-82ac06e67808

**Comment by pablorealini** [comment]
Do you know if this only happens at this level?

**Comment by Mmlh1** [comment]
No, this is general, it should happen if you move from the tower build menu onto the wave icon I believe.

**Comment by pablorealini** [comment]
Ok, thanks.
I think this is a desktop bug , right?

**Comment by Mmlh1** [comment]
Uh yeah I think so? I don't recall it happening on mobile.

**Comment by Emicon97** [comment]
This is very easy to reproduce, and this is not the only case that triggers it, as I showed in [this other issue](https://github.com/ironhidegames/issues-kr4/issues/178).

**Comment by cigumo** [comment]
duplicate of #178
